data/hails.txt

```diff
@@ -1654,6 +1764,10 @@ phrase "friendly pirate"
 		"your cargo"
 		"your ship"
 		"your fleet"
+		"your ass"
```

Comment:
This game doesn't have much profanity, as you may have noticed. "Ass" might be a bit too strong for this game's tone.

Similarly, you may notice that Captain Nate says "Get your **butt** to Mars" instead of the actual quote. :wink:

Reply:
It *should* have swears and insults, actually, but not in the modern sense. That is, Endless Sky is a thousand years in the future, so it should have slang, but it should be very different from something in the 21st century.

Reply:
consider the disabled hails; they swear at ya but they use silly insults, they don't say stuff like "If I ever see you again I'll fucking kill you!"

Reply:
Yeah, I looked around and saw words like "damn" being used as well as "hell" I figured "ass" is hardly a curse word. I honestly believe no one who knows about the internet will be offended by the word "ass" if it's a problem I'll change it, but I do agree that there should be a bit more "language" in this game. Though I am not saying we need the f word or words of similar caliber.

Reply:
"Ass" is a stronger swear word than "damn" and hell." It's not about offense, it's about taste. Too much swearing produces a puerile atmosphere that we're trying to avoid. Mature adults (such as everyone you meet in the game) don't swear all that much.

Reply:
Ass really isn't offensive man. It damn isnt offensive, I dont see why ass is. I disagree with you.

Reply:
>Mature adults (such as everyone you meet in the game) don't swear all that much.

Well we are talking about pirates here... Heck, maybe one of those teenage crew members was able to become a captain.